packages/runtime/lifecycle/src/api.ts

```diff
@@ -1,17 +1,36 @@
-import * as TE from "fp-ts/lib/TaskEither.js";
-import * as F from "fp-ts/lib";
-import { WalletAPI } from "@marlowe.io/wallet/api";
-import * as O from "fp-ts/lib/Option.js";
-import { CreateRequest, ProvideInput } from "./tx.js";
+import { WalletAPI, WalletDI } from "@marlowe.io/wallet/api";
```

Comment:
It would be good to add a package description that:
* Shows the import path (is it `@marlowe.io/runtime-lifecycle` or `@marlowe.io/runtime-lifecycle/api` ?)
* Mentions the relevant exports of the package, or at least point out that the main export is the`RuntimeLifecycle` API

<img width="973" alt="Screenshot 2023-09-25 at 11 37 50" src="https://github.com/input-output-hk/marlowe-ts-sdk/assets/2634059/48fabd52-14a7-4911-ac03-89fa9497e191">